0_portfolio_input_check_functions.R

```diff
@@ -1538,16 +1539,37 @@ add_other_to_sector_classifications <- function(audit) {
 }
 
 
-# add_bics_sector <- function(portfolio){
-#   #join in bics sectors for EQ and CB via bics_bridge
-#   bics_bridge <- read_csv("data/bics_bridge.csv")
-#   portfolio <- portfolio %>%
-#     left_join(bics_bridge, by = c("security_bics_subgroup" = "bics_subgroup"))
-#
-#   if ("bics_sector.x" %in% colnames(portfolio)) {
-#     portfolio <- portfolio %>% select(-bics_sector.x) %>% rename(bics_sector = bics_sector.y)
-#   }
-#
-# return(portfolio)
-#
-# }
+pw_calculations <- function(eq_portfolio, cb_portfolio){
+
+  port_all <- data.frame()
+
+  if(data_check(eq_portfolio)){
+
+    port_all <- bind_rows(port_all, eq_portfolio)
+  }
+
+  if(data_check(cb_portfolio)){
+
+    port_all <- bind_rows(port_all, cb_portfolio)
+
+  }
+
+  if(data_check(port_all)){
+
+  port_all <- port_all %>%  select(grouping_variables,company_id, value_usd)
```

Comment:
is `grouping_variables` the name of a column that is expected to be in `eq_portfolio` or `cb_portfolio`, or is it meant to be a character vector containing the names of columns to group by? If the latter, it should probably be unquoted, no?

Reply:
It is grouping_variables <- c("investor_name", "portfolio_name") is the latter right. What do you mean by unquoted? It is unquoted right?

Reply:
it should probably have the same syntax as [L1565](https://github.com/2DegreesInvesting/PACTA_analysis/pull/240/files#diff-9a3348aa2255d360e2e12fe80be7719715a083dc6e42d5af91fe5d53b4315b3dR1565), no?
`!!!rlang::syms(grouping_variables)`